pkg/cmd/secret/list/list_test.go

```diff
@@ -153,6 +154,30 @@ func Test_listRun(t *testing.T) {
 				"SECRET_THREE\t1975-11-30T00:00:00Z\tSELECTED",
 			},
 		},
+		{
+			name: "org tty, json",
+			tty:  true,
+			json: true,
+			opts: &ListOptions{
+				OrgName: "UmbrellaCorporation",
+			},
+			wantOut: []string{
+				// Note the `"numSelectedRepos":2` pair in the last entry.
+				`[{"name":"SECRET_ONE","numSelectedRepos":0,"selectedReposURL":"","updatedAt":"1988-10-11T00:00:00Z","visibility":"all"},{"name":"SECRET_TWO","numSelectedRepos":0,"selectedReposURL":"","updatedAt":"2020-12-04T00:00:00Z","visibility":"private"},{"name":"SECRET_THREE","numSelectedRepos":2,"selectedReposURL":"https://api.github.com/orgs/UmbrellaCorporation/actions/secrets/SECRET_THREE/repositories","updatedAt":"1975-11-30T00:00:00Z","visibility":"selected"}]`,
+			},
+		},
```

Comment:
**What level of testing do you think is appropriate when `numSelectedRepos` is not included in the fields to export?**

I'll admit this specific feature is weird from a testing perspective because we're still outputting `numSelectedRepos` even if we aren't retrieving the data.  So, it seems like there should be tests that mirror this but ensure the field is zeroed out unless it is provided as a field to export.

Reply:
Good question, and I missed testing for it. I think I should add a couple of more tests to ensure we're handling both kind of cases (i.e., with or without `numSelectedRepos`). We already have the ones that **include** the `numSelectedRepos` field. I need to add the other half. I'll push another commit with the new test cases and will ask for review again.

I think it's not weird, because if `numSelectedRepos` was not there in the `--json` argument, then we won't print it out, so no weird zeros.